### DISCUSSION TITLE
Tasmota OpenTherm command is not supported by the boiler

### DISCUSSION THREAD

**Discussion by GeorgeWatch** [arendst/Tasmota#13950] [Q&A] [answered]
Hi,
I have a boiler [Thermona EL 14 ](https://www.thermona.eu/en/electric-boilers/with-touch-display/boiler-therm-el-14)with OpenTherm +. I don't know what the differences are in the protocols. The only thing I found was the specification of [OpenTherm 2.2](https://www.domoticaforum.eu/uploaded/Ard%20M/Opentherm%20Protocol%20v2-2.pdf).
OpenTherm on [Ihor Melnyk](https://github.com/ihormelnyk/opentherm_library) worked nicely on my Arduino. Then I found Tasmota with OpenTherm. This is a better solution for HomeAssistant + mqtt.
I recompiled according to the Tasmota instructions with [OpenTherm support](https://tasmota.github.io/docs/OpenTherm/) and uploaded to WeMos D1 Mini ESP8266
After connecting tasmota OpenTherm to the boiler, Tasmota console reports an unsupported command :
[OTH]: command OEMD is not supported by the boiler. Last status: INVALID
[OTH]: command TDHW is not supported by the boiler. Last status: INVALID
[OTH]: command TOUT is not supported by the boiler. Last status: INVALID
[OTH]: command TRET is not supported by the boiler. Last status: INVALID
[OTH]: command OT12 is not supported by the boiler. Last status: INVALID
[OTH]: command OT14 is not supported by the boiler. Last status: INVALID
[OTH]: command OT19 is not supported by the boiler. Last status: INVALID
[OTH]: command OT33 is not supported by the boiler. Last status: INVALID
[OTH]: command OT116 is not supported by the boiler. Last status: INVALID
[OTH]: command OT117 is not supported by the boiler. Last status: INVALID
[OTH]: command OT120 is not supported by the boiler. Last status: INVALID
[OTH]: command OT121 is not supported by the boiler. Last status: INVALID

1. Am I doing something wrong or what protocol does Tasmota support?
2. There are so big differences between the protocols OpenTherm?

Thank you for the explanation or guidance on what can be done about it .

**Comment by kamiKAC** [comment, marked as answer]
Did you configure your tamplate correctly?
I have Ihor's interface with WemosD1 mini also. My template:
{"NAME":"OpenTherm adapter","GPIO":[1,1,320,1,4928,4960,1,1,1,1,1312,1,1,1],"FLAG":0,"BASE":18}
Some commands may be unsupported by your boiler.

**Comment by yann1420** [comment]
for instance, this sentence says
`[OTH]: command OEMD is not supported by the boiler. Last status: INVALID`
and that is exactly what is is. the command is not supported by your boiler.

My boiler, an **Intergas Prestige** also does not support all OT commands
